aws-rds-dbparametergroup/src/main/java/software/amazon/rds/dbparametergroup/BaseHandlerStd.java

```diff
@@ -189,7 +190,7 @@ private ProgressEvent<ResourceModel, CallbackContext> modifyParameters(final Pro
                                                                            final RequestLogger requestLogger) {
         ResourceModel model = progress.getResourceModel();
         CallbackContext callbackContext = progress.getCallbackContext();
-        Map<String, Parameter> parametersToModify = getModifiableParameters(model, currentDBParameters);
+        Map<String, Parameter> parametersToModify = new TreeMap<>(getModifiableParameters(model, currentDBParameters));
```

Comment:
nit: You can use https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html#toMap-java.util.function.Function-java.util.function.Function-java.util.function.BinaryOperator-java.util.function.Supplier- to avoid creating extra in-memory map

Reply:
Same comment as above.